modules/swagger-jaxrs/pom.xml

```diff
@@ -49,6 +49,26 @@
                 </configuration>
             </plugin>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>1.6</version>
```

Comment:
it possibly makes sense using latest plugin version `3.0.0`

Reply:
Fixed. Thanks!